magento/2/latest-version

```diff
@@ -1 +1 @@
-2.4.2
```

Comment:
this change has the consequence of bumping the version of magento we auto-install, let's not do that in this PR.